tests/rules/method.js

```diff
@@ -103,7 +103,14 @@ eslintTester.run("method", rule, {
         },
         {
             code: "(Async.checkAppReady = function() { return true; })();"
-        }        
+        },
+        {
+            code: "let endTime = (mapEnd || (e => e.delta))(this._data[this._data.length - 1]);",
+            parserOptions: { ecmaVersion: 6 }
+        },
+        {
```

Comment:
this is #62, as mentioned in the comment next to the new code in method.js that lets it pass.